suite-native/atoms/src/Card/CompactCardWithIconLayout.tsx

```diff
@@ -1,29 +1,25 @@
 import { ReactNode } from 'react';
 import { Pressable, PressableProps } from 'react-native';
-import { useSelector } from 'react-redux';
 
-import { selectHasRunningDiscovery } from '@suite-common/wallet-core';
-import {
-    Box,
-    Card,
-    HStack,
-    InlineAlertBox,
-    InlineAlertBoxProps,
-    Loader,
-    Text,
-    VStack,
-} from '@suite-native/atoms';
 import { Icon, IconName } from '@suite-native/icons';
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 import { Color } from '@trezor/theme';
 
+import { Card } from './Card';
+import { Box } from '../Box';
+import { InlineAlertBox, InlineAlertBoxProps } from '../InlineAlertBox/InlineAlertBox';
+import { Loader } from '../Loader';
+import { HStack, VStack } from '../Stack';
+import { Text } from '../Text';
+
 const ICON_WRAPPER_SIZE = 48;
 
 type CardVariant = 'normal' | 'danger';
-type SettingsItemCardProps = Omit<PressableProps, 'onPress'> & {
+export type CompactCardWithIconLayoutProps = Omit<PressableProps, 'onPress'> & {
     icon: IconName;
     title: ReactNode;
     subtitle: ReactNode;
+    isDisabled?: boolean;
     alertBoxProps?: Omit<InlineAlertBoxProps, 'borderRadius'>;
     onPress?: () => void;
     variant?: CardVariant;
```

Comment:
What is the benefit of omitting onPress from PressableProps and defining it again? I know this was already in develop, but anyway.
```suggestion
export type CompactCardWithIconLayoutProps = PressableProps & {
    icon: IconName;
    title: ReactNode;
    subtitle: ReactNode;
    isDisabled?: boolean;
    alertBoxProps?: Omit<InlineAlertBoxProps, 'borderRadius'>;
    variant?: CardVariant;
```

Reply:
It's a mistake, good catch. Fixed 🙈